test/spec.js

```diff
@@ -388,6 +388,66 @@ describe('checkDependencies', function () {
             });
         });
 
+        it('should check custom package name dependencies only if `checkCustomPackageNames` is true ' +
+           'and we are testing bower, not npm', function (done) {
+
+            if (packageManager !== 'npm') {
+              checkDeps({
```

Comment:
I've added the `validateIndentation` to JSCS on `master`, if you rebase your PR `npm test` will find you all these style violations.